src/pages/AddLiquidity/ConfirmAddModalBottom.tsx

```diff
@@ -25,40 +25,54 @@ export function ConfirmAddModalBottom({
   return (
     <>
       <RowBetween>
-        <TYPE.body>{currencies[Field.CURRENCY_A]?.symbol} Deposited</TYPE.body>
+        <TYPE.purple3 fontSize="14px" fontWeight={500}>
```

Comment:
The `purple3` type is a thing that has to be removed. I've added it when the theme was not yet updated, and it remained there even though it theoretically has to be removed, my bad. I think the correct type to use is `body`. It has the same color and the same props can be applied.